lib/Gitlab/Api/Environments.php

```diff
@@ -0,0 +1,33 @@
+<?php namespace Gitlab\Api;
+
+class Environments extends AbstractApi
+{
+    /**
+     * @param int $project_id
+     * @return mixed
+     */
+    public function all($project_id)
+    {
+        return $this->get($this->getProjectPath($project_id, 'environments'));
+    }
+
+    /**
+     * @param int $project_id
+     * @param array $params
+     * @return mixed
+     */
+    public function create($project_id, array $params = array())
```

Comment:
Could you use OptionsResolver here?

Reply:
I was looking at other `create()` methods and I can't see any using the `OptionsResolver` - was just going with what seemed like the convention. Will have a look and update if possible.